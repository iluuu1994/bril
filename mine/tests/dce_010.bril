@main {
  a: int = const 1;
  b: int = call a;
  b: int = call a;
  c: int = const 1;
  c: int = call a;
}
@print(x: int) {
  print a;
  ret a;
}
