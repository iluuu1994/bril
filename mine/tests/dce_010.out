@main {
  a: int = const 1;
  b: int = call a;
  b: int = call a;
  c: int = call a;
}
@print(x: int) {
  print a;
  ret a;
}
